### DISCUSSION TITLE
-define jpeg:fancy-upsampling=off works for me with convert

### DISCUSSION THREAD

**Discussion by dgilge** [ImageMagick/ImageMagick#2210] [Help]
It looks like the post https://www.imagemagick.org/discourse-server/viewtopic.php?f=3&t=27177 isn't an issue anymore. I've tried it both, with `convert` and `mogrify` and I got equal results with `identify`:

```
    Overall:
      min: 0  (0)
      max: 255 (1)
      mean: 109.641 (0.429963)
      standard deviation: 50.3596 (0.197489)
      kurtosis: -0.208038
      skewness: 0.0135179
      entropy: 0.835544
```

I'm using ImageMagick 6.9.10-23 Q16 x86_64 20190101 on Debian (Docker).

Since this bug is linked in https://www.smashingmagazine.com/2015/06/efficient-image-resizing-with-imagemagick/ which is one of the first hits when you search for image optimization in Google, I wanted to add a comment in the forum that this issue is solved. But when I click register, I get to this discussion page. So I wanted to ask if somebody who is registered there could add a comment in the IM forum that this issue is solved.

**Comment by snibgo** [comment]
Thanks for your feedback. The old forum is now locked. I am registered there, as a moderator, but I can't post to it or edit posts.
